cmd/earthly/main.go

```diff
@@ -311,7 +311,17 @@ func getVersionPlatform() string {
 }
 
 func getPlatform() string {
-	return fmt.Sprintf("%s/%s; %s", runtime.GOOS, runtime.GOARCH, osutil.GetDisplay())
+	showOSInfo := func() (info string) {
```

Comment:
Nice work-around! It's even more impressive that you contributed a fix for it upstream.

Would you be able to add a comment here mentioning something like:

```
// Work-around for windows panics; this can be removed once https://github.com/wille/osutil/pull/10 is merged
```

so we can tidy this work-around once the upstream fix is merged.

Reply:
Of course yes.  And it's done.